### PR TITLE
Add `Aetherian's Summer Singles 2` tournament article

### PR DESCRIPTION
**Pull request by AetherianFI**
Wrote the tournament article for Aetherian's Summer Singles 2 that ended a few weeks ago. My first time contributing anything here so please say if I need to fix something :3

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

File: wiki/Tournaments/AESS/en.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Aetherian's Summer Singles
+
+Index page for all of the Aetherian's Summer Singles series.
+
+- [Aetherian's Summer Singles 2](/wiki/Tournaments/AESS/2/en.md)
```
Comment (Walavouchey) [inline wiki/Tournaments/AESS/en.md:5]: put the first iteration above this one, but don't link it

File: wiki/Tournaments/AESS/en.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+<!-- List the tournaments available in the directory chronologically -->
```
Comment (Walavouchey) [inline wiki/Tournaments/AESS/en.md:7]:
```suggestion
```

File: wiki/Tournaments/AESS/2/en.md [hunk 3]
```diff
@@ -0,0 +1,405 @@
...
+tags:
+  - AESS2
+  - Aetherian's Summer Singles
+---
+
+# Aetherian's Summer Singles 2
+
+![AESS2 logo](img/mainbanner.png)
+
+The **Aetherian's Summer Singles 2** (***AESS2***) was a 1v1 double-elimination tournament hosted by ![](https://s.ppy.sh/images/flags/fi.gif) [Aetherian](https://osu.ppy.sh/users/6263040). It was the second instalment of the Aetherian's Summer Singles.
```
Comment (Walavouchey) [inline wiki/Tournaments/AESS/2/en.md:11]: this is how you do flags

```suggestion
The **Aetherian's Summer Singles 2** (***AESS2***) was a 1v1 double-elimination tournament hosted by ::{ flag=FI }:: [Aetherian](https://osu.ppy.sh/users/6263040). It was the second instalment of the Aetherian's Summer Singles.
```

File: wiki/Tournaments/AESS/2/en.md [hunk 4]
```diff
@@ -0,0 +1,405 @@
...
+| Finals | 2022-07-16/2022-07-17 |
+| Grand Finals | 2022-07-23/2022-07-24 |
+
+## Prizes
+
+| Placing | Prize(s) |
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | 12 months of osu!support, profile banner, badge |
+| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | 9 months of osu!support, profile banner |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | 6 months of osu!support, profile banner |
```
Comment (Walavouchey) [inline wiki/Tournaments/AESS/2/en.md:32]:
```suggestion
| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | 12 months of osu!supporter, profile banner, profile badge |
| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | 9 months of osu!supporter, profile banner |
| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | 6 months of osu!supporter, profile banner |
```

File: wiki/Tournaments/AESS/2/en.md [hunk 5]
```diff
@@ -0,0 +1,405 @@
...
+| *4th place* | 4 months of osu!supporter |
+| *5th-6th place* | 2 months of osu!supporter |
+| *7th-8th place* | 1 month of osu!supporter |
```
Comment (Walavouchey) [inline wiki/Tournaments/AESS/2/en.md:35]:
```suggestion
| *5th–6th place* | 2 months of osu!supporter |
| *7th–8th place* | 1 month of osu!supporter |
```

File: wiki/Tournaments/AESS/2/en.md [hunk 6]
```diff
@@ -0,0 +1,405 @@
...
+| Custom Mapper | ![](https://s.ppy.sh/images/flags/au.gif) [VP757](https://osu.ppy.sh/users/9451871), ![](https://s.ppy.sh/images/flags/us.gif) [Kurashina Asuka](https://osu.ppy.sh/users/7476493) |
```
Comment (Walavouchey) [inline wiki/Tournaments/AESS/2/en.md:49]:
```suggestion
| Mappool tester | ![](https://s.ppy.sh/images/flags/id.gif) [Crezz](https://osu.ppy.sh/users/7108275), ![](https://s.ppy.sh/images/flags/kr.gif) [Nopekjk](https://osu.ppy.sh/users/4585186), ![](https://s.ppy.sh/images/flags/no.gif) [Naigo](https://osu.ppy.sh/users/10640486), ![](https://s.ppy.sh/images/flags/ua.gif) [dench](https://osu.ppy.sh/users/10073635), ![](https://s.ppy.sh/images/flags/pl.gif) [Mastasz](https://osu.ppy.sh/users/1876565), ![](https://s.ppy.sh/images/flags/fi.gif) [Eliaasi](https://osu.ppy.sh/users/9690571), ![](https://s.ppy.sh/images/flags/my.gif) [Zeph2003](https://osu.ppy.sh/users/10343292), ![](https://s.ppy.sh/images/flags/se.gif) [scylla](https://osu.ppy.sh/users/9405745), ![](https://s.ppy.sh/images/flags/fi.gif) [Haadez](https://osu.ppy.sh/users/8925266), ![](https://s.ppy.sh/images/flags/de.gif) [criller](https://osu.ppy.sh/users/8116659), ![](https://s.ppy.sh/images/flags/ru.gif) [gamer228666](https://osu.ppy.sh/users/5981005), ![](https://s.ppy.sh/images/flags/pl.gif) [HEDAYO](https://osu.ppy.sh/users/9323821) |
| Custom mapper | ![](https://s.ppy.sh/images/flags/au.gif) [VP757](https://osu.ppy.sh/users/9451871), ![](https://s.ppy.sh/images/flags/us.gif) [Kurashina Asuka](https://osu.ppy.sh/users/7476493) |
```

File: wiki/Tournaments/AESS/2/en.md [hunk 7]
```diff
@@ -0,0 +1,405 @@
...
+| Streamer | ![](https://s.ppy.sh/images/flags/ca.gif) [Drou](https://osu.ppy.sh/users/415932), ![](https://s.ppy.sh/images/flags/de.gif) [Dragoncurve](https://osu.ppy.sh/users/6675367) |
+| Commentator | ![](https://s.ppy.sh/images/flags/ca.gif) [Drou](https://osu.ppy.sh/users/415932), ![](https://s.ppy.sh/images/flags/us.gif) [-Koda](https://osu.ppy.sh/users/12260184), ![](https://s.ppy.sh/images/flags/fi.gif) [Death Grips](https://osu.ppy.sh/users/10632187), ![](https://s.ppy.sh/images/flags/fi.gif) [Nev-](https://osu.ppy.sh/users/11836334) |
+| Graphics | ![](https://s.ppy.sh/images/flags/ca.gif) [Zyoulou](https://osu.ppy.sh/users/8668722), ![](https://s.ppy.sh/images/flags/fi.gif) [shdewz](https://osu.ppy.sh/users/10000899) |
```
Comment (Walavouchey) [inline wiki/Tournaments/AESS/2/en.md:52]:
```suggestion
| Graphic designer | ![](https://s.ppy.sh/images/flags/ca.gif) [Zyoulou](https://osu.ppy.sh/users/8668722), ![](https://s.ppy.sh/images/flags/fi.gif) [shdewz](https://osu.ppy.sh/users/10000899) |
```

File: wiki/Tournaments/AESS/2/en.md [hunk 8]
```diff
@@ -0,0 +1,405 @@
...
+| 25-32 | ![](https://s.ppy.sh/images/flags/uk.gif) [Mahmood](https://osu.ppy.sh/users/7627844), ![](https://s.ppy.sh/images/flags/se.gif) [TOnyrino](https://osu.ppy.sh/users/7338010), ![](https://s.ppy.sh/images/flags/de.gif) [Tim Kackner](https://osu.ppy.sh/users/14385814), ![](https://s.ppy.sh/images/flags/ro.gif) [Kehest](https://osu.ppy.sh/users/6145000), ![](https://s.ppy.sh/images/flags/us.gif) [Olympia](https://osu.ppy.sh/users/5357992), ![](https://s.ppy.sh/images/flags/fi.gif) [Nev-](https://osu.ppy.sh/users/11836334), ![](https://s.ppy.sh/images/flags/us.gif) [Gabey](https://osu.ppy.sh/users/12904237), ![](https://s.ppy.sh/images/flags/us.gif) [Raoul](https://osu.ppy.sh/users/9924405) |
```
Comment (Walavouchey) [inline wiki/Tournaments/AESS/2/en.md:68]:
```suggestion
| 1–8 | ![](https://s.ppy.sh/images/flags/hk.gif) [mcy4](https://osu.ppy.sh/users/2165650), ![](https://s.ppy.sh/images/flags/kr.gif) [[Karcher]](https://osu.ppy.sh/users/11443437), ![](https://s.ppy.sh/images/flags/ro.gif) [SatonoDiamond](https://osu.ppy.sh/users/12366071), ![](https://s.ppy.sh/images/flags/us.gif) [Swiq_](https://osu.ppy.sh/users/11269055), ![](https://s.ppy.sh/images/flags/pl.gif) [maliszewski](https://osu.ppy.sh/users/12408961), ![](https://s.ppy.sh/images/flags/us.gif) [WillCookie](https://osu.ppy.sh/users/6404488), ![](https://s.ppy.sh/images/flags/us.gif) [Jacob3](https://osu.ppy.sh/users/13628898), ![](https://s.ppy.sh/images/flags/us.gif) [Jakson](https://osu.ppy.sh/users/8788058) |
| 9–16 | ![](https://s.ppy.sh/images/flags/us.gif) [Venpris](https://osu.ppy.sh/users/8861593), ![](https://s.ppy.sh/images/flags/us.gif) [-Koda](https://osu.ppy.sh/users/12260184), ![](https://s.ppy.sh/images/flags/de.gif) [rundyyy](https://osu.ppy.sh/users/10917620), ![](https://s.ppy.sh/images/flags/ca.gif) [noncycle](https://osu.ppy.sh/users/12701607), ![](https://s.ppy.sh/images/flags/je.gif) [rudj](https://osu.ppy.sh/users/11592896), ![](https://s.ppy.sh/images/flags/us.gif) [EzChock](https://osu.ppy.sh/users/9276293), ![](https://s.ppy.sh/images/flags/us.gif) [BoshyMan741](https://osu.ppy.sh/users/4830687), ![](https://s.ppy.sh/images/flags/us.gif) [DarkScrap](https://osu.ppy.sh/users/13241562) |
| 17–24 | ![](https://s.ppy.sh/images/flags/cl.gif) [Danidesu](https://osu.ppy.sh/users/2748187), ![](https://s.ppy.sh/images/flags/il.gif) [Galog](https://osu.ppy.sh/users/7799629), ![](https://s.ppy.sh/images/flags/pl.gif) [gnahus](https://osu.ppy.sh/users/12779141), ![](https://s.ppy.sh/images/flags/nz.gif) [Saiyku](https://osu.ppy.sh/users/13767572), ![](https://s.ppy.sh/images/flags/de.gif) [aahoff](https://osu.ppy.sh/users/11371245), ![](https://s.ppy.sh/images/flags/sg.gif) [200201envenho](https://osu.ppy.sh/users/8152145), ![](https://s.ppy.sh/images/flags/id.gif) [-Reuto-](https://osu.ppy.sh/users/10717635), ![](https://s.ppy.sh/images/flags/sg.gif) [GSBlank](https://osu.ppy.sh/users/2312106) |
| 25–32 | ![](https://s.ppy.sh/images/flags/uk.gif) [Mahmood](https://osu.ppy.sh/users/7627844), ![](https://s.ppy.sh/images/flags/se.gif) [TOnyrino](https://osu.ppy.sh/users/7338010), ![](https://s.ppy.sh/images/flags/de.gif) [Tim Kackner](https://osu.ppy.sh/users/14385814), ![](https://s.ppy.sh/images/flags/ro.gif) [Kehest](https://osu.ppy.sh/users/6145000), ![](https://s.ppy.sh/images/flags/us.gif) [Olympia](https://osu.ppy.sh/users/5357992), ![](https://s.ppy.sh/images/flags/fi.gif) [Nev-](https://osu.ppy.sh/users/11836334), ![](https://s.ppy.sh/images/flags/us.gif) [Gabey](https://osu.ppy.sh/users/12904237), ![](https://s.ppy.sh/images/flags/us.gif) [Raoul](https://osu.ppy.sh/users/9924405) |
```

File: wiki/Tournaments/AESS/2/en.md [hunk 9]
```diff
@@ -0,0 +1,405 @@
...
+
+See the [Spreadsheet](https://docs.google.com/spreadsheets/d/1ESDw34gBfnDWwUrXEPZNxsi4MN89p156aIYP4RYbuYs/edit#gid=0) for a list of all participants.
```
Comment (Walavouchey) [inline wiki/Tournaments/AESS/2/en.md:70]:
```suggestion
See [this spreadsheet](https://docs.google.com/spreadsheets/d/1ESDw34gBfnDWwUrXEPZNxsi4MN89p156aIYP4RYbuYs/edit#gid=0) for a list of all participants.
```

File: wiki/Tournaments/AESS/2/en.md [hunk 10]
```diff
@@ -0,0 +1,405 @@
...
+  1. [Fujikawa Chiai - Gomi no Hi (a_Blue) [Extra]](https://osu.ppy.sh/beatmapsets/1235121#osu/2729414)
+  2. [BAND-MAID - Spirit!! (Lasse) [Expert!!]](https://osu.ppy.sh/beatmapsets/1306578#osu/2709014)
+- HardRock
+  1. [BUD VIRGIN LOGIC - x Senritsu -Schlehit Melodie- (Mordred) [Lasse's Bud Light Extra]](https://osu.ppy.sh/beatmapsets/1569807#osu/3217086)
+  2. [ak+q - Axium Crisis (MrSergio) [Collapse]](https://osu.ppy.sh/beatmapsets/812551#osu/1704329)
+- DoubleTime
+  1. [S3RL - Candy (feat. Sara) (Nuvolina) [Nuvolina & Anto's Insane]](https://osu.ppy.sh/beatmapsets/829956#osu/1742403)
+  2. [Akiyama Uni - The Grimoire of Alice (Hollow Wings)[Insane]](https://osu.ppy.sh/beatmapsets/123839#osu/344619)
+
+
```
Comment (Walavouchey) [inline wiki/Tournaments/AESS/2/en.md:256]:
```suggestion
```

File: wiki/Tournaments/AESS/2/en.md [hunk 11]
```diff
@@ -0,0 +1,405 @@
...
+## Match results
+
+### Grand Finals
+
+2022-07-23/2022-07-24
```
Comment (Walavouchey) [inline wiki/Tournaments/AESS/2/en.md:261]: did the match take place across two days?

have a look at [an example](https://osu.ppy.sh/wiki/en/Tournaments/OWC/2021#match-results) for the date formatting here

File: wiki/Tournaments/AESS/2/en.md [hunk 12]
```diff
@@ -0,0 +1,405 @@
...
+| **SatonoDiamond** | **5** | 1 | Nev- | [#1](https://osu.ppy.sh/community/matches/101545295) |
+| **EzChock** | **5** | 1 | gnahus | [#1](https://osu.ppy.sh/community/matches/101607694) |
+| **WillCookie** | **5** | 4 | Tim Kackner | [#1](https://osu.ppy.sh/community/matches/101603892) |
+| rundyyy | 3 | **5** | **200201envenho** | [#1](https://osu.ppy.sh/community/matches/101543829) |
+
+## Ruleset
+
+### General
+
+- This tournament is a 1v1 double-elimination osu!standard tournament.
```
Comment (Walavouchey) [inline wiki/Tournaments/AESS/2/en.md:368]: it's officially just the "osu!" game mode

```suggestion
- This tournament is a 1v1 double-elimination osu! tournament.
```

File: wiki/Tournaments/AESS/2/en.md [hunk 13]
```diff
@@ -0,0 +1,405 @@
...
+- All matches will use Score V2 and NF will be enforced on every map.
+- All players have to treat staff members and all other players wiith high level of respect. Any kind of mistreatment can results into a being banned from the tournament.
```
Comment (Walavouchey) [inline wiki/Tournaments/AESS/2/en.md:370]:
```suggestion
- All players have to treat staff members and all other players with a high level of respect. Any kind of mistreatment may result in being banned from the tournament.
```

File: wiki/Tournaments/AESS/2/en.md [hunk 14]
```diff
@@ -0,0 +1,405 @@
...
+- This is an open rank tournament.
+- Staff members are not allowed to participate in the tournament.
+- Every player is required to join the tournament discord server.
```
Comment (Walavouchey) [inline wiki/Tournaments/AESS/2/en.md:373]:
```suggestion
- Every player is required to join the tournament Discord server.
```

File: wiki/Tournaments/AESS/2/en.md [hunk 15]
```diff
@@ -0,0 +1,405 @@
...
+
+- There will be a new mappool every week.
+- Mappools follow the following format:
+    - Qualifiers: 4xNM, 2xHD, 2xHR, 2xDT
+    - Round of 32: 5xNM, 3xHD, 3xHR, 3xDT, 1xTB
+    - Round of 16: 5xNM, 3xHD, 3xHR, 3xDT, 1xTB
+    - Quarterfinals: 6xNM, 4xHD, 4xHR, 4xDT, 1xTB
+    - Semifinals: 6xNM, 4xHD, 4xHR, 4xDT, 1xTB
+    - Finals: 6xNM, 4xHD, 4xHR, 4xDT, 1xTB
+    - Grand Final: 6xNM, 4xHD, 4xHR, 4xDT, 1xTB
```
Comment (Walavouchey) [inline wiki/Tournaments/AESS/2/en.md:386]:
```suggestion

| Stage | Maps |
| :-- | :-- |
| Qualifiers | 4 NM, 2 HD, 2 HR, 2 DT |
| Round of 32 | 5 NM, 3 HD, 3 HR, 3 DT, 1 TB |
| Round of 16 | 5 NM, 3 HD, 3 HR, 3 DT, 1 TB |
| Quarterfinals | 6 NM, 4 HD, 4 HR, 4 DT, 1 TB |
| Semifinals | 6 NM, 4 HD, 4 HR, 4 DT, 1 TB |
| Finals | 6 NM, 4 HD, 4 HR, 4 DT, 1 TB |
| Grand Final | 6 NM, 4 HD, 4 HR, 4 DT, 1 TB |

```

File: wiki/Tournaments/AESS/2/en.md [hunk 16]
```diff
@@ -0,0 +1,405 @@
...
+ - Every mappool will be a slightly harder than the previous one. Star Rating goal for Grand Finals mappool is 8.0*
```
Comment (Walavouchey) [inline wiki/Tournaments/AESS/2/en.md:387]:
```suggestion
 - Every mappool will be slightly harder than the previous one. Maps in the Grand Finals mappool will have 8 stars.
```

File: wiki/Tournaments/AESS/2/en.md [hunk 17]
```diff
@@ -0,0 +1,405 @@
...
+
+### Qualifiers
+
+- Qualifier mappool consists of 4xNM, 2xHD, 2xHR & 2x DT maps.
```
Comment (Walavouchey) [inline wiki/Tournaments/AESS/2/en.md:391]:
```suggestion
- The Qualifier mappool consists of 4 NM, 2 HD, 2 HR, and 2 DT maps.
```

File: wiki/Tournaments/AESS/2/en.md [hunk 18]
```diff
@@ -0,0 +1,405 @@
...
+- Qualifier mappool will be played in following order NM -> DT.
```
Comment (Walavouchey) [inline wiki/Tournaments/AESS/2/en.md:392]:
```suggestion
- The Qualifier mappool will be played in following order: NM -> DT.
```

File: wiki/Tournaments/AESS/2/en.md [hunk 19]
```diff
@@ -0,0 +1,405 @@
...
+- NF and Score V2 will be used on every map.
+- Mappool will be played once only.
```
Comment (Walavouchey) [inline wiki/Tournaments/AESS/2/en.md:394]:
```suggestion
- The mappool will be played once only.
```

File: wiki/Tournaments/AESS/2/en.md [hunk 20]
```diff
@@ -0,0 +1,405 @@
...
+- If players disconnects during a map they can replay it after all other maps have been played and other players have left the lobby.
```
Comment (Walavouchey) [inline wiki/Tournaments/AESS/2/en.md:395]:
```suggestion
- If a player disconnects during a map they can replay it after all other maps have been played and all other players have left the lobby.
```

File: wiki/Tournaments/AESS/2/en.md [hunk 21]
```diff
@@ -0,0 +1,405 @@
...
+- Players will be seeded following this pattern: Seed 1 VS Seed 32, Seed 2 VS Seed 31, ...
```
Comment (Walavouchey) [inline wiki/Tournaments/AESS/2/en.md:396]:
```suggestion
- Players will be matched against each other in the pattern of *seed 1 vs. seed 32*, *seed 2 vs. seed 31*, etc.
```

File: wiki/Tournaments/AESS/2/en.md [hunk 22]
```diff
@@ -0,0 +1,405 @@
...
+
+### Match procedure
+
+- All matches will use NF and Score V2.
```
Comment (Walavouchey) [inline wiki/Tournaments/AESS/2/en.md:400]: it's like the third time this is mentioned

File: wiki/Tournaments/AESS/2/en.md [hunk 23]
```diff
@@ -0,0 +1,405 @@
...
+- Players have 10 minute grace period to join the lobby after an agreed time. If 10 minutes pass and player is still not in the lobby, the other player wins the match.
```
Comment (Walavouchey) [inline wiki/Tournaments/AESS/2/en.md:401]:
```suggestion
- Players have a 10-minute grace period to join the lobby after an agreed time. If 10 minutes pass and a player is still not in the lobby, the other player wins the match.
```

File: wiki/Tournaments/AESS/2/en.md [hunk 24]
```diff
@@ -0,0 +1,405 @@
...
+- Players are allowed to pick 1 map for warm up. Maximum length for warm up is 4 minutes. Warm ups have to be in good taste.
```
Comment (Walavouchey) [inline wiki/Tournaments/AESS/2/en.md:402]:
```suggestion
- Players are allowed to pick 1 map for warmup, with a maximum length of 4 minutes. Warmups have to be in good taste.
```

File: wiki/Tournaments/AESS/2/en.md [hunk 25]
```diff
@@ -0,0 +1,405 @@
...
+- After warm ups both players will use !roll command to decide pick and ban order. Winner of the roll can decide pick or ban order, the loser of the roll will decide the remaining order.
```
Comment (Walavouchey) [inline wiki/Tournaments/AESS/2/en.md:403]:
```suggestion
- After warmups both players will use the `!roll` command to decide pick and ban order. The winner of the roll can decide pick or ban order, and the loser of the roll will decide the remaining order.
```

**Review by Walavouchey — CHANGES_REQUESTED** [review]

File: wiki/Tournaments/AESS/2/en.md [hunk 26]
```diff
@@ -0,0 +1,407 @@
+---
+tags:
+  - AESS2
+  - Aetherian's Summer Singles
+---
+
+# Aetherian's Summer Singles 2
+
+![AESS2 logo](img/mainbanner.png)
```
Comment (Walavouchey) [inline wiki/Tournaments/AESS/2/en.md:9]: rename the file

```suggestion
![AESS2 logo](img/banner.png)
```

Comment (AetherianFI) [reply wiki/Tournaments/AESS/2/en.md]: fixed

File: wiki/Tournaments/AESS/2/en.md [hunk 27]
```diff
@@ -0,0 +1,413 @@
...
+
+### Semifinals
+Detailed statistics for this round can be found [here](https://docs.google.com/spreadsheets/d/1_G1Nl-bDAJCUrPfiEsYCUn3AsXlVdJe9Mw4P4_tziQc/edit#gid=795207119).
+
+2022-07-09/2022-07-10
+
+| Player 1 |  |  | Player 2 | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| **GSBlank** | **6** | 1 | Saiyku | [#1](https://osu.ppy.sh/community/matches/102127956) |
+| **Swiq_** | **0** | -1 | Jakson | ***forfeit*** |
```
Comment (Walavouchey) [inline wiki/Tournaments/AESS/2/en.md:289]: no need to bold "*forfeit*"

**Comment by AetherianFI** [comment]
image
Am I supposed to do something when this happens?

File: wiki/Tournaments/AESS/2/en.md [hunk 28]
```diff
@@ -0,0 +1,483 @@
...
+### Finals
+
+Detailed statistics for this round can be found [here](https://docs.google.com/spreadsheets/d/1Fu7K7d0XDyhL7bk5t0DxWSOihBUWojMxLYkEOAP8QIE/edit#gid=795207119).
+
+Saturday, July 16, 2022
+
+| Player 1 |  |  | Player 2 | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| **mcy4** | **7** | 3 | -Reuto- | [#1](https://osu.ppy.sh/community/matches/102310621) |
+| WillCookie | 1 | **7** | **Swiq_** | [#1](https://osu.ppy.sh/community/matches/102321669) |
```
Comment (TPGPL) [inline wiki/Tournaments/AESS/2/en.md:284]:
```suggestion
| WillCookie | 1 | **7** | **Swiq\_** | [#1](https://osu.ppy.sh/community/matches/102321669) |
```

File: wiki/Tournaments/AESS/2/en.md [hunk 29]
```diff
@@ -0,0 +1,483 @@
...
+
+Sunday, July 17, 2022
+
+| Player 1 |  |  | Player 2 | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| Swiq_ | 0 | **7** | **mcy4** | [#1](https://osu.ppy.sh/community/matches/102341595) |
```
Comment (TPGPL) [inline wiki/Tournaments/AESS/2/en.md:290]:
```suggestion
| Swiq\_ | 0 | **7** | **mcy4** | [#1](https://osu.ppy.sh/community/matches/102341595) |
```

File: wiki/Tournaments/AESS/2/en.md [hunk 30]
```diff
@@ -0,0 +1,483 @@
...
+| **maliszewski** | **7** | 2 | [Karcher] | [#1](https://osu.ppy.sh/community/matches/102337887) |
+
+### Semifinals
+Detailed statistics for this round can be found [here](https://docs.google.com/spreadsheets/d/1_G1Nl-bDAJCUrPfiEsYCUn3AsXlVdJe9Mw4P4_tziQc/edit#gid=795207119).
+
+Friday, July 8, 2022
```
Comment (TPGPL) [inline wiki/Tournaments/AESS/2/en.md:296]:
```suggestion
Friday, July 8, 2022:
```

File: wiki/Tournaments/AESS/2/en.md [hunk 31]
```diff
@@ -0,0 +1,483 @@
...
+| Player 1 |  |  | Player 2 | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| mcy4 | 4 | **6** | **maliszewski** | [#1](https://osu.ppy.sh/community/matches/102105872) |
+
+Saturday, July 9, 2022
+
+| Player 1 |  |  | Player 2 | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| **GSBlank** | **6** | 1 | Saiyku | [#1](https://osu.ppy.sh/community/matches/102127956) |
+| **Swiq_** | **0** | -1 | Jakson | *forfeit* |
```
Comment (TPGPL) [inline wiki/Tournaments/AESS/2/en.md:307]:
```suggestion
| **Swiq\_** | **0** | -1 | Jakson | *forfeit* |
```

File: wiki/Tournaments/AESS/2/en.md [hunk 32]
```diff
@@ -0,0 +1,483 @@
...
+| **-Reuto-** | **0** | -1 | rundyyy | *forfeit* |
+| **[Karcher]** | **6** | 0 | WillCookie | [#1](https://osu.ppy.sh/community/matches/102120655) |
+
+Sunday, July 10, 2022
+
+| Player 1 |  |  | Player 2 | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| EzChock | 3 | **6** | **noncycle** | [#1](https://osu.ppy.sh/community/matches/102145896) |
+| **Swiq_** | **6** | 2 | GSBlank | [#1](https://osu.ppy.sh/community/matches/102161740) |
```
Comment (TPGPL) [inline wiki/Tournaments/AESS/2/en.md:316]:
```suggestion
| **Swiq\_** | **6** | 2 | GSBlank | [#1](https://osu.ppy.sh/community/matches/102161740) |
```

File: wiki/Tournaments/AESS/2/en.md [hunk 33]
```diff
@@ -0,0 +1,483 @@
...
+| **Gabey** | **0** | -1 | Galog | *forfeit* |
+
+Saturday, June 25, 2022
+
+| Player 1 |  |  | Player 2 | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| **Kehest** | **5** | 4 | aahoff | [#1](https://osu.ppy.sh/community/matches/101763787) |
+| **mcy4** | **5** | 2 | DarkScrap | [#1](https://osu.ppy.sh/community/matches/101763792) |
+| Mahmood | 3 | **5** | **GSBlank** | [#1](https://osu.ppy.sh/community/matches/101767442) |
+| **Swiq_** | 0 | **-1** | rudj | *forfeit* |
```
Comment (TPGPL) [inline wiki/Tournaments/AESS/2/en.md:376]:
```suggestion
| **Swiq\_** | 0 | **-1** | rudj | *forfeit* |
```

File: wiki/Tournaments/AESS/2/en.md [hunk 34]
```diff
@@ -0,0 +1,483 @@
...
+
+Saturday, June 18, 2022
+
+| Player 1 |  |  | Player 2 | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| **mcy4** | **5** | 1 | Raoul | [#1](https://osu.ppy.sh/community/matches/101560106) |
+| **DarkScrap** | **5** | 3 | Danidesu | [#1](https://osu.ppy.sh/community/matches/101579995) |
+| Jakson | 3 | **5** | **Mahmood** | [#1](https://osu.ppy.sh/community/matches/101578040) |
+| Venpris | -1 | **0** | **GSBlank** | *forfeit* |
+| **Swiq_** | **5** | 3 | Em_Rald | [#1](https://osu.ppy.sh/community/matches/101581530) |
```
Comment (TPGPL) [inline wiki/Tournaments/AESS/2/en.md:425]:
```suggestion
| **Swiq\_** | **5** | 3 | Em\_Rald | [#1](https://osu.ppy.sh/community/matches/101581530) |
```

File: wiki/Tournaments/AESS/2/en.md [hunk 35]
```diff
@@ -0,0 +1,483 @@
...
+| maliszewski | **5** | 0 | noncycle | [#1](https://osu.ppy.sh/community/matches/101767849) |
+| SatonoDiamond | -1 | **0** | **EzChock** | *forfeit* |
+| **WillCookie** | **5** | 3 | 200201envenho | [#1](https://osu.ppy.sh/community/matches/101753576) |
+
+Sunday, June 26, 2022
+
+| Player 1 |  |  | Player 2 | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| **Raoul** | **5** | 1 | Danidesu | [#1](https://osu.ppy.sh/community/matches/101802795) |
+| Em_Rald | 3 | **5** | **Saiyku** | [#1](https://osu.ppy.sh/community/matches/101779305) |
```
Comment (TPGPL) [inline wiki/Tournaments/AESS/2/en.md:386]:
```suggestion
| Em\_Rald | 3 | **5** | **Saiyku** | [#1](https://osu.ppy.sh/community/matches/101779305) |
```

**Review by TPGPL — CHANGES_REQUESTED** [review]

**Review by TPGPL — APPROVED** [review]
next time please use feature branches for contributions, not your `master` branch

**Comment by AetherianFI** [comment]
> next time please use feature branches for contributions, not your `master` branch

oh my mb, i never use github so i dont have good idea how everything works here lol

**Pull request merged**